src/pages/index.tsx

```diff
@@ -0,0 +1,3 @@
+import Homepage from "../modules/homepage";
```

Comment:
use `@` everywhere for consistency

Reply:
As this is just one folder back using `@` feels like an overkill.